documents/operating.md

```diff
@@ -52,14 +52,223 @@ SECTION 4: 💵💵💵 MONEY 💵💵💵
+Sessions, which are to be scheduled and announced by the Overmind on
+April 1st of every year.  The schedule can be updated as needed, but
+a notice of 10 days must be given before any Plotting Session can be
+held.
+
+April 1st marks the beginning of the governing year, and officer elections
+are to be held during the month of April.
+
+
+### Deliberation Processes
+#### Plotting Session
+Plotting Sessions are agenda driven and require a Quorum.  They
+are led by the chair (as determined by the chain of command).
+
+The Hive Mind will use Robert's Rules of Order to run Plotting Sessions
+in an orderly and Robertly fashion.
+
+When possible, video should be enabled for any plotting session.  Votes should be
+cast using physical emoji 👍, 👎, and 😐 (or any skin tone variation) depicted
+on signs attached to sticks and held up to indicate votes of yes, no, and abstain
+respectively.  Members can also vote by raising their hands, but this will
+be considered lame.
+
+A role call vote can take place instead if the Secretary of Lore or the meeting
+chair deems it necessary in order to properly tally a vote.
+
+
+#### GitHub
+Decisions made on GitHub are either interim decisions that must
+culminate in a Plotting Session to finalize, or they are decisions
+related to BIFFUD's GitHub process and documentation policy.
+
+Almost all decisions involve GitHub at some level.
+
+The method of deliberation is similar in both cases:
+
+1. The person requesting a vote should review the instructions set
+forth in `CONTRIBUTING.md` based on the type of vote being requested.
+
+2. A Pull Request must be made against the `master` branch of the
+BIFFUD corporate repository.
+
+3. The Keeper of Code reviews the Pull Request and requests any changes
+around standards violations or mistakes.
+
+4. Once the Pull Request is deemed valid, the Keeper of Code requests
+a review from all Corporate Overlords.
+
+4. Corporate Overlords vote using GitHub's review feature.  Approval
+is a yes vote, Rejection is a no vote, and Comments are used to request
+more information or changes.
+
+5. Requests that require a Plotting Session are left open until a final
+vote is taken during a plotting session.  The Secretary of Lore updates
+the Pull Request with a record of the vote outcome from the Plotting
+Session.
+
+6. Requests that do not require a Plotting Session are left open until
+they pass or fail.  Votes that do not require a Plotting Session
+automatically fail if they have been open for over a month.
+
+If a vote is considered preliminary, although actions may be triggered
+by initial indications of passage, the vote is not final until after a
+Plotting Session has been held.
+
+If a vote is final and passes, the Pull Request is merged by the Keeper
+of Code and the branch is deleted.
+
+If a vote is final and fails, the Pull Request is closed without merge
+by the Keeper of Code and the branch is deleted.
+
+
+### Voting Decisions
```

Comment:
Does it make sense to actually list this stuff out?  Is this too overwhelming?

I'll take a minute to see what other bylaws lay out for this kind of thing -- maybe it's not necessary to list the kinds of things we would vote for.